src/core/Akka.Cluster.Tests/ClusterSpec.cs

```diff
@@ -83,8 +86,10 @@ public void A_cluster_must_publish_initial_state_as_events_to_subscribers()
         {
             try
             {
+                // TODO: this should be removed
```

Comment:
Probably, we shouldn't. Looks like the scala test using result from the previous tests, that's why they don't call joinSeedNodes again